src/modules/UI/scenes/SendConfirmation/SendConfirmation.ui.js

```diff
@@ -61,19 +65,19 @@ type routerParam = {
 
 type Props = SendConfirmationStateProps & SendConfirmationDispatchProps & routerParam
 
-type State = {
+type State = {|
```

Comment:
@wolverineks What is point of the pipe syntax?

Reply:
its a tighter / stricter flow annotation
i think it provides better flow feedback, and could prevent some bugs